### PR TITLE
W-17670895 sf RNs: Aug 7

### PR DESCRIPTION
**Pull request by jshackell-sfdc**

### CODE REVIEW

File: releasenotes/README.md [hunk 1]
```diff
@@ -26,12 +26,28 @@ Additional documentation:
 * [Salesforce CLI Setup Guide](https://developer.salesforce.com/docs/atlas.en-us.sfdx_setup.meta/sfdx_setup/sfdx_setup_intro.htm)
 
 
-## 2.53.6 (August 7, 2024) [stable-rc]
+## 2.54.6 (August 14, 2024) [stable-rc]
+
+**ANNOUNCEMENT:** If you still use the deprecated `sfdx`-style commands, be sure you read [this pinned GitHub issue](https://github.com/forcedotcom/cli/issues/2974) about the upcoming removal commands of these commands:  `force:source:*`, `force:org:create`, and `force:org:delete`.
```
Comment (shetzel) [inline releasenotes/README.md]:
```suggestion
**ANNOUNCEMENT:** If you still use the deprecated `sfdx`-style commands, be sure you read [this pinned GitHub issue](https://github.com/forcedotcom/cli/issues/2974) about the upcoming removal of these commands:  `force:source:*`, `force:org:create`, and `force:org:delete`. 
```

File: releasenotes/README.md [hunk 2]
```diff
@@ -26,12 +26,28 @@ Additional documentation:
...
+
+* FIX: The `project deploy start` command now correctly handles deployments of changed custom fields, even when the parent object hasn't changed. (source-deploy-retrieve PR [#1375](https://github.com/forcedotcom/source-deploy-retrieve/pull/1375))
+
+* FIX: The `project deploy start` command now correctly handles deployments of objects whose source files are spread across multiple folders in your project. (GitHub issue [#2230](https://github.com/forcedotcom/cli/issues/2230), source-deploy-retrieve PR [#1381](https://github.com/forcedotcom/source-deploy-retrieve/pull/1381))
+
+## 2.53.6 (August 7, 2024) [stable]
+
+**ANNOUNCEMENT:** If you still use the deprecated `sfdx`-style commands, be sure you read [this pinned GitHub issue](https://github.com/forcedotcom/cli/issues/2974) about the upcoming removal commands of these commands:  `force:source:*`, `force:org:create`, and `force:org:delete`.
```
Comment (shetzel) [inline releasenotes/README.md]:
```suggestion
**ANNOUNCEMENT:** If you still use the deprecated `sfdx`-style commands, be sure you read [this pinned GitHub issue](https://github.com/forcedotcom/cli/issues/2974) about the upcoming removal of these commands:  `force:source:*`, `force:org:create`, and `force:org:delete`. 
```

**Review by shetzel — CHANGES_REQUESTED** [review]

File: releasenotes/README.md [hunk 3]
```diff
@@ -26,12 +26,28 @@ Additional documentation:
 * [Salesforce CLI Setup Guide](https://developer.salesforce.com/docs/atlas.en-us.sfdx_setup.meta/sfdx_setup/sfdx_setup_intro.htm)
 
 
-## 2.53.6 (August 7, 2024) [stable-rc]
+## 2.54.6 (August 14, 2024) [stable-rc]
+
+**ANNOUNCEMENT:** If you still use the deprecated `sfdx`-style commands, be sure you read [this pinned GitHub issue](https://github.com/forcedotcom/cli/issues/2974) about the upcoming removal of these commands:  `force:source:*`, `force:org:create`, and `force:org:delete`.
```
Comment (mshanemc) [inline releasenotes/README.md]: I think the `sfdx-style` verbage is confusing/misleading.  Many commands will still work by typing `sfdx` or using colons like `force:org:list` or using  flags like `--skipconnectionstatus` 

It's more about whether you use one of the deprecated commands we're planning to remove.

and the removal also affects `force:mdapi` commands!

Comment (jshackell-sfdc) [reply releasenotes/README.md]: thanks!  will update

**Pull request merged**